components/Icons/DAOIcon.tsx

```diff
@@ -0,0 +1,12 @@
+import Icon from '@components/layouts/Icon';
+import { ClassNameProps } from '@utils/interface/global-interface';
+
+export default function DAOIcon({ className }: ClassNameProps) {
+  return (
+    <Icon className={className} viewBox="0 0 512 512">
+      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 576 512">
```

Comment:
Did this work ok? Coz the `Icon` component is basically a `svg` wrapper. You won't need the svg anymore, just the path.